selenium/integration/computeprofile.spec.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { BASE_URL } from '../support/constants';
+import {buildChromeDriver, loginIfRequired, makeBackendRequest, makeElementHelpers} from '../support/utils';
+
+async function deleteProfile(path) {
+  await makeBackendRequest({
+    path: `${path}/disable`,
+    method: 'POST',
+  });
+
+  await makeBackendRequest({
+    path,
+    method: 'DELETE'
+  });
+}
+
+describe('Compute Profile', () => {
+  let driver;
+  let se;
+
+  beforeAll(async () => {
+    driver = buildChromeDriver();
+    se = makeElementHelpers(driver);
+    await loginIfRequired(driver);
+  });
+
+  afterAll(async () => {
+    await driver.quit()
+  });
+
+  // Due to special handling of system namespace, we need to explicitly test navigating into system namespace
+  // profile pages
+  it('should be able to create system compute profile', async () => {
+    await driver.get(`${BASE_URL}/cdap/ns/system/profiles/create`);
+    await se.waitByTestId('provisioner-gcp-dataproc');
+
+    await se.findByTestId('provisioner-gcp-dataproc').click();
+    await se.waitByTestId('profile-create-btn');
+
+    const createButton = await se.findByTestId('profile-create-btn');
+    expect(createButton.isEnabled()).toBeTruthy();
```

Comment:
Not sure if this test has the same issue, but for my `disabled` undoButton case, `.isEnabled()` returns `{}` and it's truthy. I can only either check if the element has attribute `disabled` or assert that `expect(undoButton.isEnabled()).not.toBe(true);`

https://stackoverflow.com/a/44564830

Reply:
Ok I have to add await before `isEnabled()`, otherwise it returns a Promise and is always truthy

Reply:
^ I wonder if it should be changed to `const createButtonEnabled = await se.findByTestId('profile-create-btn').isEnabled();`

Reply:
Correct, I recommend we don't use `.toBeTruthy()`